packages/terra-table/src/proptypes/cellShape.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+const cellShape = PropTypes.shape({
+  /**
+   * Content that will be rendered within the Cell.
+   */
+  content: PropTypes.node,
+  /**
+   *  Boolean indicating if cell contents are masked.
+   */
+  isMasked: PropTypes.bool,
+  /**
+   * Boolean value indicating whether or not the column header is selectable.
+   */
+  isSelectable: PropTypes.bool,
+  /**
+   * Boolean value indicating whether or not the cell should render as selected.
+   */
+  isSelected: PropTypes.bool,
```

Comment:
Not needed

Reply:
This will be needed because it is reused by the grid components.  However, we do not have to publicize in the Table API documentation.